packages/core/package.json

```diff
@@ -50,6 +50,7 @@
     ]
   },
   "dependencies": {
+    "deepmerge": "^4.0.0",
```

Comment:
change to peerDependency

Reply:
after discussion, will leave this for now (do not want to force every app to install deepmerge)